nms/app/packages/magmalte/alerts/cwfAlerts.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {prom_alert_config} from '@fbcnms/magma-api';
+
+export const cwfAlerts: {[string]: prom_alert_config} = {
+  'Container Restarts': {
+    alert: 'Container Restarts',
+    expr:
+      'time() - container_start_time_seconds{name=~".+",name!="cadvisor"} < 300',
+    labels: {severity: 'major'},
+    for: '6m',
+    annotations: {
+      description:
+        'A container has been running for less than 5 minutes, for more than 6 minutes',
```

Comment:
While this description is accurate, it might be better to say "This container appears to be restarting often" or something like that. Also I think you can add label expansions into the description like this: "The container {{ <container_id_label }} is restarting often"